contents/blog/the-posthog-array-1-24-0.mdx

```diff
@@ -0,0 +1,350 @@
+
+Oh, and we're giving away some awesome [PostHog merch](https://merch.posthog.com) as a thank you!
+
+## Community Shoutouts
+
+Big thanks to the following members of our community who have contributed to PostHog over this release cycle:
+
+- [adrienbrault](https://github.com/adrienbrault)
+- [jeduden](https://github.com/jeduden)
+- [leggetter](https://github.com/leggetter)
+- [rethab](https://github.com/rethab)
+- [lharress](https://github.com/lharress)
+- [jeduden](https://github.com/jeduden)
+- [bard](https://github.com/bard)
+- [jonathanclarke](https://github.com/jonathanclarke)
+- [shogunpurple](https://github.com/shogunpurple)
+- [samcaspus](https://github.com/samcaspus)
+- [well-balanced](https://github.com/well-balanced)
+- [gempain](https://github.com/gempain)
+- [bronsonavila](https://github.com/bronsonavila)
+- [joesaunderson](https://github.com/joesaunderson)
+- [wushaobo](https://github.com/wushaobo)
+
+
+## Open Roles
+
+If you'd love to help us build PostHog, we have a lot of openings available!
+
+We're looking for:
+
+- Full Stack Engineers
+- Senior Full Stack Engineers
+- Sales Engineer
+- Senior Mobile Engineer
+- Senior Product Designer
+- Product Marketer
+- Any role you think you'd be a great fit for
+
+Check out our [Careers page](https://posthog.com/careers) for more info.
+
+
+## Bug Fixes and Performance Improvements
+
+In addition to the highlights listed above, we also merged a bunch of PRs improving PostHog's performance and fixing bugs:
+
+- Update plugin server to 0.16.10 [\#3976](https://github.com/PostHog/posthog/pull/3976) ([posthog-bot](https://github.com/posthog-bot))
+- Rename redefined variables [\#3975](https://github.com/PostHog/posthog/pull/3975) ([mariusandra](https://github.com/mariusandra))
+- Three small UX fixes [\#3972](https://github.com/PostHog/posthog/pull/3972) ([Twixes](https://github.com/Twixes))
+- Fix persons search [\#3971](https://github.com/PostHog/posthog/pull/3971) ([yakkomajuri](https://github.com/yakkomajuri))
+- Update plugin server to 0.16.9 [\#3970](https://github.com/PostHog/posthog/pull/3970) ([posthog-bot](https://github.com/posthog-bot))
+- Fix display of plugin error seconds [\#3965](https://github.com/PostHog/posthog/pull/3965) ([mariusandra](https://github.com/mariusandra))
+- Update plugin server to 0.16.8 [\#3959](https://github.com/PostHog/posthog/pull/3959) ([posthog-bot](https://github.com/posthog-bot))
+- Update plugin server to 0.16.7 [\#3958](https://github.com/PostHog/posthog/pull/3958) ([posthog-bot](https://github.com/posthog-bot))
+- Remove fixed typescript errors [\#3956](https://github.com/PostHog/posthog/pull/3956) ([paolodamico](https://github.com/paolodamico))
+- Update plugin server to 0.16.6 [\#3955](https://github.com/PostHog/posthog/pull/3955) ([posthog-bot](https://github.com/posthog-bot))
+- Update plugin server to 0.16.5 [\#3954](https://github.com/PostHog/posthog/pull/3954) ([posthog-bot](https://github.com/posthog-bot))
```

Comment:
Maybe we can strip out all these and instead show PRs from the plugin-server repo as well?

Reply:
I generally do a clean up of some stuff here but it's always a bit hard and arbitrary. 

e.g. I remove reverts, contributors bot, etc. 

I was debating removing these too but a version bump is a bit more important I guess

idk the solution - ultimately we have so many repos we can include too

Reply:
I'll leave this in for now, but happy to rethink the approach for the next release.